plugins/legion-hid/README.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Plugin: Legion HID
+---
+
+## Introduction
+
+The Legion HID plugin is used for interacting with the MCU on Legion Go 2 devices.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+a packed binary file format.
+
+This plugin supports the following protocol ID:
+
+* `com.lenovo.legion-hid`
```

Comment:
@hughsie can you get this added to LVFS?  I think it's agreed at this point (and was the name we reserved for this device with GoS taking `com.lenovo.legion-hid2`)